dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -230,16 +231,21 @@ const merchandisingAdStyles = css`
 
 const inlineAdStyles = css`
 	position: relative;
+	background-color: ${schemedPalette('--ad-background-article-inner')};
```

Comment:
I am not fond of the long name, but there does need to be a distinction as on some pages e.g. liveblongs the right and inline ads have darker backgrounds.

something with an alpha channel might be better instead of seperate variables. But would lead to coloured ad backgrounds on opinion articles...